src/main/resources/languages/languages_de_DE.properties

```diff
@@ -31,7 +31,7 @@ BRIGHTNESS_SENSOR=Helligkeitssensor
 BUTTON=Schalter
 DIMMER=Dimmer
 HANDLE=Griffkontakt
-MOTION_DETECTOR=Bewegungssensor
+MOTION_DETECTOR=Bewegungssensorlanguages_de_DE.properties
```

Comment:
Do you really want to add the "languages_de_DE.properties" suffix? ;)

Reply:
Well, it was cherry-picked from https://github.com/openbase/bco.bcozy/commit/35a432e83eb3bbd16b6fabaa66e3db4fb07d8575 ;)